src/notifications/payment.ts

```diff
@@ -29,7 +29,7 @@ export const transactionNotification = async ({
   let title
 
   if (satsPrice) {
-    const usd = (amount * satsPrice!).toFixed(2)
+    const usd = (amount * satsPrice).toFixed(2)
```

Comment:
What is the implication of removing the `!` here?

Reply:
`!` after a variable is a typescript directive to avoid type checking

Reply:
In this particular case, it's not significant since there is a `satsPrice` check.